bitcoin/wallet.go

```diff
@@ -24,9 +24,22 @@ type BitcoinWallet interface {
 	// Get the confirmed and unconfirmed balances
 	Balance() (confirmed, unconfirmed int64)
 
+	// Check if we have sufficient funds to make a tx
+	CheckSuffientFunds(amount int64, feeLevel spvwallet.FeeLevel) error
```

Comment:
Couldn't this be done outside the interface with something like `Balance() - amount - fee`? Keeping the interfaces as small as possible is good for allowing other implementations of the interface to be written easier.